web/components/Nav/Nav.tsx

```diff
@@ -15,36 +17,60 @@ interface NavProps {
 export const Nav = ({ onFrontPage, currentPath, ticketsUrl }: NavProps) => {
   const [menuOpened, setMenuOpened] = useState(false);
   const contentsId = 'nav-menu-contents';
+  const [menuItemsCount, setMenuItemsCount] = useState(0);
 
   useEffect(() => {
     document.body.classList.toggle('main-menu-open', menuOpened);
+    if (!menuOpened) {
+      setMenuItemsCount(0);
+    }
   }, [menuOpened]);
 
   const toggleMenu = () => setMenuOpened(!menuOpened);
   const closeMenu = () => setMenuOpened(false);
 
-  const BasicMenuItem = ({
-    urlPath,
-    label,
-  }: {
-    urlPath: string;
-    label: string;
-  }) => (
-    <li>
-      <Link href={urlPath}>
-        <a
-          className={clsx(
-            styles.link,
-            urlPath === currentPath && styles.current
-          )}
-          onClick={closeMenu}
-        >
-          {label}
-        </a>
-      </Link>
-    </li>
+  const menuItems = [
+    <MenuItem
+      key={1}
+      {...{ currentPath, closeMenu }}
+      href={ticketsUrl}
+      label="Tickets"
+      className={styles.ticketItem}
+    />,
+    <MenuItem
+      key={2}
+      {...{ currentPath, closeMenu }}
+      href="/program"
+      label="Program"
+    />,
+    <MenuItem
+      key={3}
+      {...{ currentPath, closeMenu }}
+      href="/sponsorship-information"
+      label="Sponsorship"
+    />,
+    <MenuItem
+      key={4}
+      {...{ currentPath, closeMenu }}
+      href="/registration-info"
+      label="Registration"
+    />,
+    <MenuItem
+      key={5}
+      {...{ currentPath, closeMenu }}
+      href="/about"
+      label="About"
+    />,
+  ];
+
+  useInterval(
```

Comment:
Have you considered setting an inline `animation-delay` style instead (with increasing value for each menu item)? And adding `visibility: hidden` / `visibility: visible` to the keyframes (not completely sure if it's sufficient to just use the `from` and `to` keyframes or if there should be an extra keyframe with a really low percentage to ensure that the `visible` part triggers righ away). The `animation-fill-mode` property can be used to ensure the link stays hidden during the delay interval.

If this is possible, then we would
- avoid the extra code from `useInterval`
- avoid mutating the DOM during the animation
- avoid having to re-define the interval length and the media query here

Reply:
Taking a cue from the navblock animation PR, the inline style could alternatively be a custom property (e.g. `--delay`), for better decoupling. Presumably pretty straightforward to set up in such a way that it falls back gracefully (i.e. just shows the menu items without animating) in IE11 which would drop the CSS declaration(s) that use that property.

Reply:
This is a _great_ suggestion, thank you. 28e906b.